data-processing-lib/python/src/data_processing/data_access/data_access_factory.py

```
@@ -29,6 +28,10 @@ class DataAccessFactory(DataAccessFactoryBase):
     Data Access class based on these parameters.
     This class has to be serializable, so that we can pass it to the actors
     """
+    default_class = 'DataAccessLocal'
+    default_module = 'data_processing.data_access'
```

the name "default_module" can be somewhat misleading since it's actually expecting the name of the package. Something along the lines of "default_package" could be more clear.